composer.json

```diff
@@ -23,7 +23,12 @@
     "require-dev": {
         "symfony/mime": "^5.4 || ^6.0 || ^7.0",
         "symfony/phpunit-bridge": "^v6.4.1 || ^7.0.1",
-        "symfony/security-core": "^5.4 || ^6.0 || ^7.0"
+        "symfony/security-core": "^5.4 || ^6.0 || ^7.0",
+        "phpstan/phpstan": "^1.10",
+        "phpstan/phpstan-deprecation-rules": "^1.1",
+        "phpstan/phpstan-phpunit": "^1.3",
+        "phpstan/phpstan-symfony": "^1.3",
+        "phpstan/phpstan-strict-rules": "^1.5"
```

Comment:
Please put them before the symfony packages to keep them sorted.

I also suggest adding the config section to tell composer to sort packages (I generally add it in all my new projects but this one has not been updated to have it)